lib/vehicle.rb

```diff
@@ -0,0 +1,36 @@
+class Vehicle 
+  attr_reader :year,
+              :make,
+              :model,
+              :passengers,
+              :speeding
+
+  def initialize(year, make, model)
+    # require 'pry'; binding.pry
+    @year = year
+    @make = make
+    @model = model 
+    @passengers = []
+    @speeding = false
+
+  end
+
+
+  def speed 
+    @speeding = true
+  end
+
+  def speeding?
+    return true if @speeding == true
+    false
+  end
+
+  def add_passenger(passenger)
+    @passengers << passenger
+    @passengers
```

Comment:
```suggestion
    @passengers << passenger
```
`@passengers << passenger` will automatically return the `@passengers` array so there's no need to manually return it again at the end.